SearchValueControllerTest.cls

```diff
@@ -33,6 +33,8 @@ private class SearchValueControllerTest {
         incontroller.searchString = 'test';
         incontroller.explore();
         incontroller.searchString.length();
+        
+        system.assertNotEquals(0,incontroller.searchString.length());
```

Comment:
These are wrong asserts instead you should have assert for result which will return after `explore` method call